lib/flux-core/src/lib.rs

```diff
@@ -1,5 +1,7 @@
 #![no_std]
+#![feature(step_trait)]
```

Comment:
We have to put this under a cfg, otherwise it won't work for non-Flux builds.

```suggestion
#![cfg_attr(flux, feature(step_trait))]
```